recipes/sqlite_orm/all/conanfile.py

```diff
@@ -68,3 +68,8 @@ def package(self):
     def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "SqliteOrm")
         self.cpp_info.set_property("cmake_target_name", "sqlite_orm")
```

Comment:
maybe remove `self.cpp_info.set_property("cmake_target_name", "sqlite_orm")` to not break CMakeDeps when `cmake_target_name` will switch to absolute name.